cmake/OpenCVUtils.cmake

```diff
@@ -1768,3 +1768,19 @@ macro(ocv_git_describe var_name path)
     set(${var_name} "unknown")
   endif()
 endmacro()
+
+function(ocv_update_file filepath content)
+  if(EXISTS "${filepath}")
+    file(READ "${filepath}" __content)
+  else()
+    set(__content "")
+  endif()
+  if("${__content}" STREQUAL "${content}")
+    if(";${ARGN};" MATCHES ";VERBOSE;")
```

Comment:
It would be great to document this function:
```.cmake
# ocv_update_file(filepath content [VERBOSE])
#   write content to file 
#   will not change modification time in case when file already exists and content has not changed
```